### PR TITLE
November 2024 changelog

### PR DESCRIPTION
**Pull request by adriendupuis**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | N/A
| Versions      | ~All~ master, 4.6
| Edition       | All

- Added what's new in the doc during November 2024, based on d7833506240f8dc54c108d256627703d62876d1f...c636be472cab8f702d82f6ce3b0b2273beb6e916.
    - Nota bene: I choose to not mention [`ibexa:user:update-user` command](https://doc.ibexa.co/en/master/users/update_basic_user_data/)
- Added what's new on 4.6.14's PHP API from #2568
- Minor changes in previous months for consistency.

Preview: https://ez-systems-developer-documentation--2564.com.readthedocs.build/en/2564/resources/new_in_doc/ and its left nav tree.

#### Checklist

- [ ] Text renders correctly
- [ ] Text has been checked with vale
- [ ] Description metadata is up to date
- [ ] Redirects cover removed/moved pages
- [ ] Code samples are working
- [ ] PHP code samples have been fixed with PHP CS fixer
- [ ] Added link to this PR in relevant JIRA ticket or code PR

### CODE REVIEW

File: docs/resources/new_in_doc.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## November 2024
+
+- [AI Actions documentation](ai_actions.md)
+- [New `ibexa:user:update-user` command to manage some basic user data from CLI](update_basic_user_data.md)
+- [New `IsBookmarked` location criterion](isbookmarked_criterion.md)
+- [`IsUserEnabled` is available on Solr and Elastisearch](isuserenabled_criterion.md)
+- `4.6.14` and `3.3.41` are out, and documentation is updated accordingly.
+- When searching from the top bar, you may see, at the bottom of the drop-down suggestion list, a link to a page to see more results.
```
Comment (adriendupuis) [inline docs/resources/new_in_doc.md:16]: TODO:
- Sort in subsections
- Manage the "new" pills

**Review by mnocon — APPROVED** [review]
Thank you!

File: docs/resources/new_in_doc.md [hunk 2]
```diff
@@ -6,6 +6,26 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
+- Added [AI Actions documentation](https://doc.ibexa.co/en/master/ai_actions/ai_actions/)
+
+### Search
+
+- Added [new `IsBookmarked` location criterion](https://doc.ibexa.co/en/master/search/criteria_reference/isbookmarked_criterion/)
+- [`IsUserEnabled` is now available on Solr and Elastisearch](https://doc.ibexa.co/en/master/search/criteria_reference/isuserenabled_criterion/)
+
+### Documentation
+
+- When searching from the top bar, you may see, at the bottom of the drop-down suggestion list, a link to a page to see more results.
```
Comment (adriendupuis) [inline docs/resources/new_in_doc.md:27]:
```suggestion
- When searching from the top bar, you may see, at the bottom of the drop-down suggestion list, a link to a page to see more results.

### PHP API

- Added the following new namespaces, interfaces, and classes to the public PHP API:
    - [`Ibexa\Contracts\Core\Validation`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/namespaces/ibexa-contracts-core-validation.html) namespace and its descendants
    - [`Ibexa\Contracts\Notifications\SystemNotification`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/namespaces/ibexa-contracts-notifications-systemnotification.html) namespace and its descendants
    - [`Ibexa\Contracts\Notifications\Value\Recipent\UserRecipientInterface`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/classes/Ibexa-Contracts-Notifications-Value-Recipent-UserRecipientInterface.html)
    - [`Ibexa\Contracts\ProductCatalog\Values\Product\Query\Criterion\UpdatedAt`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/classes/Ibexa-Contracts-ProductCatalog-Values-Product-Query-Criterion-UpdatedAt.html)
    - [`Ibexa\Contracts\ProductCatalog\Values\Product\Query\Criterion\UpdatedAtRange`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/classes/Ibexa-Contracts-ProductCatalog-Values-Product-Query-Criterion-UpdatedAtRange.html)
    - [`Ibexa\Contracts\ProductCatalog\ProductReferencesResolverStrategy`](https://doc.ibexa.co/en/master/api/php_api/php_api_reference/classes/Ibexa-Contracts-ProductCatalog-ProductReferencesResolverStrategy.html)
```

File: docs/resources/new_in_doc.md [hunk 3]
```diff
@@ -6,6 +6,36 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
```
Comment (julitafalcondusza) [inline docs/resources/new_in_doc.md:27]:
```suggestion
- When you search using the top bar, you can see a link to a page with further results at the bottom of the drop-down suggestion list.
```

File: docs/resources/new_in_doc.md [hunk 4]
```diff
@@ -6,6 +6,36 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
+
+### PHP API
+
+- Added the following new namespaces, interfaces, and classes to the public PHP API:
```
Comment (julitafalcondusza) [inline docs/resources/new_in_doc.md:31]: You use "Added (...)" so I think that it's not necessary to add "new" in this sentence. 
It kind of means the same here.

```suggestion
- Added the following namespaces, interfaces, and classes to the public PHP API:
```

File: docs/resources/new_in_doc.md [hunk 5]
```diff
@@ -6,6 +6,36 @@ description: Overview of major recent additions to Ibexa DXP documentation.
...
+- Added [v4.6.13 to v4.6.14 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.6/update_from_4.6/#v4614) which include security fixes.
+- Added [v3.3.40 to v3.3.41 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_3.3/update_from_3.3/#v3341) which include security fixes.
+
+### Content management
+
+- Added [AI Actions documentation](https://doc.ibexa.co/en/master/ai_actions/ai_actions/)
+
+### Search
+
+- Added [new `IsBookmarked` location criterion](https://doc.ibexa.co/en/master/search/criteria_reference/isbookmarked_criterion/)
```
Comment (julitafalcondusza) [inline docs/resources/new_in_doc.md:22]:
```suggestion
- New [`IsBookmarked` location criterion](https://doc.ibexa.co/en/master/search/criteria_reference/isbookmarked_criterion/)
```

**Review by julitafalcondusza — COMMENTED** [review]
See my comments.

File: docs/resources/new_in_doc.md (hunk 3 above)
Comment (adriendupuis) [reply docs/resources/new_in_doc.md:27]: @julitafalcondusza The link is not always there. Maybe I should add in which condition it appears:
```suggestion
- When you search using the top bar, if there is more than the 10 listed results, you can see a link to a page with further results at the bottom of the drop-down suggestion list.
```

Comment (julitafalcondusza) [reply docs/resources/new_in_doc.md:27]: "If there **are** ..."

**Review by julitafalcondusza — APPROVED** [review]

**Review by dabrt — APPROVED** [review]

**Pull request merged**